src/components/OrderDetails/index.js

```diff
@@ -127,10 +128,19 @@ export const OrderDetails = (props) => {
         setOrderState({ ...orderState, order: result, loading: false })
       }
       if (error) {
-        const message = Array.isArray(result) ? result[0] : typeof result === 'string' ? result : 'INTERNAL_ERROR'
-        const defaultMessage = message !== 'INTERNAL_ERROR' ? message : t('INTERNAL_ERROR', 'Internal Error')
-
-        setOrderState({ ...orderState, error: [defaultMessage], loading: false })
+        setForceUpdate(null)
+        if (result[0] == 'outside pickup area, insert reasons to force update') {
```

Comment:
@carlosg-ordering  Please read about new ESx, use `===` instead `==`


You have 2 equals conditions. You can improve with:

```javascript
const deliveryMessages: {
  delivery: {
    text: 'outside delivery area, insert reasons to force update',
    value: 11
  },
  pickup: {
    text: 'outside pickup area, insert reasons to force update',
    value: 9
  },
}
```

With this obj you can validate and use only one set for the state `setOrderState` and `setForceUpdate`